include/seqan/sequence/string_set_dependent_tight.h

```diff
@@ -91,7 +91,7 @@ struct Dependent;
 
 // StringSet with individual sequences in a tight string of string pointers and corr. IDs
 template <typename TString>
-class StringSet<TString, Dependent<Tight> >
+class /*[[deprecated]]*/ StringSet<TString, Dependent<Tight> >
```

Comment:
The tight dependent stringset is deprecated but it is still the default and used in several applications. One cannot change the default to generic since it introduces several warnings and failed tests and as far as I can tell will require quite the amount of work to change. But it should become deprecated as soon as possible (hence the comment).